app/src/main/java/com/fstyle/androidtrainning/data/local/storage/ExternalData.java

```diff
@@ -7,6 +7,7 @@
 import android.graphics.BitmapFactory;
 import android.net.Uri;
 import android.provider.MediaStore;
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports